### PR TITLE
add base url for openai client

### PR DESCRIPTION
**Pull request by hawktang**
add base url for openai client to support customer url in future

### CODE REVIEW

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/53f06026e8d4ad95fdeb890b11e8e629eae50bbc is created.

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/041ddf81370e3b76016cc5f4d2b030b4fba45264 is created.

File: adala/runtimes/_openai.py [hunk 1]
```diff
@@ -167,7 +167,10 @@ class OpenAIChatRuntime(Runtime):
 
     @computed_field
     def _client(self) -> OpenAI:
-        return OpenAI(api_key=self.openai_api_key)
+        return OpenAI(
+            api_key=self.openai_api_key,
+            base_url=DEFAULT_CREATE_COMPLETION_URL
```
Comment (niklub) [inline adala/runtimes/_openai.py:172]: Does the base URL refer to the completion endpoint but not something like `https://api.openai.com/v1` ?

**Review by niklub — CHANGES_REQUESTED** [review]

**Comment by hawktang** [comment]
Hi Niklub,

It refer to something like https://api.openai.com/v1/

Allow to change it future can make adala support models through apps like
litellm proxy.

Peter Ze TANG (汤赜)

**Comment by matt-bernstein** [comment]
Hi @hawktang , thanks for the PR! Apologies for the long delay, but we've updated this change and merged it in https://github.com/HumanSignal/Adala/pull/145

**Comment by robot-ci-heartex** [comment]
Commit https://github.com/HumanSignal/infra/commit/dfdd6b61ae77de50a8a42732626c1bc29651e675 is created.

**Pull request closed without merging**